charts/vela-core/templates/kubevela-controller.yaml

```diff
@@ -135,6 +135,7 @@ spec:
             - "--enable-cluster-gateway"
             {{ end }}
             - "--application-re-sync-period={{ .Values.controllerArgs.reSyncPeriod }}"
+            - "--concurrent-reconciles={{ .Values.concurrentReconciles }}"
```

Comment:
So, this is a mistake that we forget to set args while we have this value in chart?  @Somefive @yangsoon

Reply:
Seems like a bug. We have this parameter in values.yaml but not use it. This change looks reasonable for me. /cc @yangsoon

Reply:
this seems forget to set, LGTM